x/faucet/types/genesis.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+// GenesisState - all faucet state that must be provided at genesis
+type GenesisState struct {
+	// TODO: The amount of fund that a user withdrew
+}
+
+// TODO : Implement the genesis functions
+// NewGenesisState creates a new GenesisState object
```

Comment:
I thinks its better to keep make the params such as epoch time ,and max value of token allowed to withdraw as part of the genesis .  It is much more secure that way.

Reply:
Can this be added in asana and left for testnet?